src/containers/ConversionForm/style.css

```diff
@@ -29,36 +38,27 @@
     margin-right: 20px;
 }
 
-.container button {
-    margin-right: 20px;
-    min-width: 80px;
+.upload {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
 }
 
-.container span svg path {
-    /* this makes the down arrow in the select box visible  */
-    color: var(--light-theme-btn-default-color);
+.container :global(.ant-upload-span) {
+    background-color: var(--overlay-purple);
 }
 
-.title {
-    font-size: 30px;
-    padding-bottom: 0px;
-    margin-bottom: 0em;
-    font-weight: 600;
-    letter-spacing: .05rem;
+.form-content button {
```

Comment:
in general, I would be careful about ever styling a pure html element. Since this was `.container button` it was styling everything within the conversion form (which the processing overlay was a child of) so the margin right was getting applied to your buttons